src/Tag/MediaSequenceTag.php

```diff
@@ -15,7 +15,7 @@ class MediaSequenceTag extends AbstractTag
 {
     use SingleValueTagTrait;
 
-    private $mediaSequence = 0;
```

Comment:
Hi,
Is there a reason for removing the initial value of 0 for $mediaSequence property?
It results in the media sequence of the first segment to be null if the playlist does not contain a #EXT-X-MEDIA-SEQUENCE tag and it is not desired according to HLS RFC 
https://tools.ietf.org/html/rfc8216#section-4.3.3.2 \:
"If the Media Playlist file does not contain an EXT-X-MEDIA-SEQUENCE tag, then the Media Sequence Number of the first Media Segment in the Media Playlist SHALL be considered to be 0."

Reply:
Well.. if I recall correctly, I thought it was more appropriate to let someone choose whether a 'EXT-X-MEDIA-SEQUENCE' line will be generated when dumping a m3u8 object, so `null` means don't generate the line. and I thought it should be the default behavior.

However I've seen the problem as in your case, so I think this lib needs a bug-fix. thank you for the report :)

Reply:
Thanks Chris for fast response :-)

Reply:
Sure it is :)

As you've already know how to fix it, why not create a PR ?

Reply:
Will do :-)